sdk/file_sdk/Common/tag.cpp

```diff
@@ -12,26 +13,6 @@ static const string kExtendedPrefix = "Extended";
 
 namespace mip {

```

Comment:
you need to make sure that all changes here are will be in the final upe+rms branch

Reply:
? aren't they suppose to move the code here?
when they do, i will remove this class and use theirs

---
In reply to: [136713188](https://github.com/AzureAD/rms-sdk-for-cpp/pull/140#discussion_r136713188) [](ancestors = 136713188)